### ISSUE TITLE
v1.10.1 seems to cause issues when blacklisting an addon

### ISSUE THREAD

**Issue by bertdeblock** [ef4/ember-auto-import#344]
Hi! 👋

I was testing out a new addon in one of our projects and came across an issue.

The addon in question can be found [here](https://github.com/bagaar/ember-graphql-mocking).
It has `ember-auto-import` defined as a normal dependency.

The addon is only needed for writing acceptance tests so I wanted to blacklist it from the host app.
When doing so I noticed a "missing module" error.

I have set up a reproduction repo here: [ember-graphql-mocking-vs-ember-auto-import](https://github.com/bertdeblock/ember-graphql-mocking-vs-ember-auto-import).

Steps to reproduce:
- Clone the reproduction repo
- Install modules
- Run `ember s`
- You should see the following error: `Could not find module "fast-memoize" imported from "ember-intl/-private/formatters/-format-datetime"`
- Next, downgrade `ember-auto-import` to `v1.10.0`
- Reinstall modules
- Run `ember s` again
- You should see `9/29/1990` printed on your screen, which means now everything works as expected

Not blacklisting the addon or removing `ember-auto-import` as a dependency also fixes the "missing module" error.

I've also came to the conclusion that maybe the addon doesn't actually need `ember-auto-import`, but I wanted to make an issue nonetheless just in case.

**Comment by ef4** [comment]
Yes, thanks, I think this was addressed by ember-auto-import 2.0.
